package.json

```
@@ -50,6 +50,7 @@
     "@types/styled-components": "^4.1.18",
     "@typescript-eslint/eslint-plugin": "^1.11.0",
     "@typescript-eslint/parser": "^1.11.0",
+    "@urql/devtools": "../urql-devtools-exchange/",
```

Can we use the name from the npm tarball? We can yarn link if needed

---

This needs to be changed but won't work until a new devtools package is released with the types.

```suggestion
    "@urql/devtools": "~0.0.3",
```

---

Let's make work of that release, I like the split up a lot